### PR TITLE
Create danser-go.json

### LINKED ISSUE #3186: [Request] Danser
### Criteria

For a package to be acceptable in the Main bucket, it should be:

- [X] a non-GUI tool
- [X] a reasonably well-known and widely used developer tool
- [X] the latest stable version of the program
- [X] the full version i.e. not a trial version
- [X] a fairly standard install (e.g. uses a version-specific download URL, no elaborate pre/post install scripts)

### Information

**Name:** danser-go

**Description:** Dancing visualizer of osu! standard maps and custom osu! client written in Go. Also a generator for osu! videos. 

**Homepage:** https://github.com/Wieku/danser-go

**Download link(s):** https://github.com/Wieku/danser-go/releases

**Some indication of popularity/repute:** 
* 421 stars on GitHub
* Multiple tutorials with tens of thousand of views, and satisfying videos using it that have millions ([1](https://www.youtube.com/c/Wieku/videos?view=0&sort=p&flow=grid), [2](https://www.youtube.com/watch?v=0l7Bt8Vo7dc), [3](https://www.youtube.com/watch?v=7gUaxpeQG5g))
* [Discord with 3K members](https://discord.com/invite/UTPvbe8)

I have a manifest ready for it, let me know if this fits in Main or if I should move to Extras

### PR DESCRIPTION
**Pull request by couleurm**
Close #3186 
- [X] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/danser-go.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "version": "0.6.7",
+  "description": "Dancing visualizer of osu! standard maps and custom osu! client written in Go. Also a generator for osu! videos.",
+  "homepage": "https://github.com/Wieku/danser-go",
+  "license": "MIT",
+  "architecture": {
+    "64bit": {
+      "url": "https://github.com/Wieku/danser-go/releases/download/0.6.8/danser-0.6.8-win.zip#dl.7z",
```
Comment (rashil2000) [inline bucket/danser-go.json:8]:
```suggestion
      "url": "https://github.com/Wieku/danser-go/releases/download/0.6.8/danser-0.6.8-win.zip",
```

Already an archive, so not required

File: bucket/danser-go.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      "hash": "b54b43df55f09a239e55078759e8c84ed085d8dd4a436f34fa6faae5db095580"
+    }
+  },
+  "bin": "danser.exe",
+  "persist": "settings",
```
Comment (rashil2000) [inline bucket/danser-go.json:13]: Is this a file or a folder?

**Review by rashil2000 — CHANGES_REQUESTED** [review]
Reformat the manifest to use 4 spaces as tab instead of 2

File: bucket/danser-go.json (hunk 2 above)
Comment (couleurm) [reply bucket/danser-go.json:13]: > Is this a file or a folder?

settings is a folder created when first executing danser which holds all the json configuration files

**Pull request merged**